Gemfile

```diff
@@ -17,6 +17,10 @@ group :test do
   gem "selenium-webdriver", "~> 4"
 end
 
+group :development, :test do
+  gem "appraisal", github: "thoughtbot/appraisal"
```

Comment:
I'm targeting `main` for now, but we'll be able to use the [next version](https://github.com/thoughtbot/appraisal/issues/204) when it's released.